jquery.xblock.js

```diff
@@ -299,10 +299,29 @@
             return cookieOptions;
         },
 
+        get_querystring: function(){
+            // For easier test stubbing.
+            return location.search;
+        },
+
         init: function(options, root) {
             var $this = this,
                 deferred = $.Deferred(),
-                blockURL = this.getViewUrl('student_view', options);
+                blockURL = this.getViewUrl('student_view', options),
+                data = options.data;
+
+            // http://stackoverflow.com/a/901144/882918
+            function getParameterByName(name) {
+                name = name.replace(/[\[]/, "\\[").replace(/[\]]/, "\\]");
+                var regex = new RegExp("[\\?&]" + name + "=([^&#]*)"),
+                    results = regex.exec($this.get_querystring());
+                return results === null ? "" : decodeURIComponent(results[1].replace(/\+/g, " "));
+            }
+
+            var activateChild = getParameterByName('activate');
```

Comment:
Can you pass this explicitly as an option (and update the README), instead of fetching it from the current page location, which is specific to apros? Someone trying to use this would have a hard time figuring out how to make this work otherwise.

Actually, it's already possible to pass data to the instantiation call - can you directly use this? That would remove the need for code changes in jquery-xblock, you would only need to add info about this in the README.